src/login/travel-document-onboarding/NoTravelDocument.tsx

```diff
@@ -0,0 +1,73 @@
+import {useTranslation} from '@atb/translations';
+import {useNavigation} from '@react-navigation/native';
+import {View} from 'react-native';
+import ThemeText from '@atb/components/text';
+import TravelDocumentOnboardingTexts from '@atb/translations/screens/subscreens/TravelDocumentOnboarding';
+import Button from '@atb/components/button';
+import {StyleSheet} from '@atb/theme';
+import React from 'react';
+import {StaticColorByType} from '@atb/theme/colors';
+import {Crash} from '@atb/assets/svg/color/images';
+
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+export function NoTravelDocument({}: {}): JSX.Element {
+  const styles = useThemeStyles();
+  const {t} = useTranslation();
+  const navigation = useNavigation();
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.topView}>
+        <ThemeText
+          type="body__primary--jumbo--bold"
+          style={styles.alignCenter}
+          color={themeColor}
+        >
+          {t(TravelDocumentOnboardingTexts.error.heading)}
+        </ThemeText>
+      </View>
+      <View style={styles.svgContainer}>
+        <Crash width="185px" height="185px" />
```

Comment:
I would update this svg and add the correct once after https://github.com/AtB-AS/design-system/pull/46 is approved and merged and a new version of our design system is published.